src/libraries/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -874,7 +874,7 @@ private bool ConsumeStringNextSegment()
                                             for (; j < localBuffer.Length; j++)
                                             {
                                                 byte nextByte = localBuffer[j];
-                                                if (!JsonReaderHelper.IsHexDigit(nextByte))
+                                                if (!char.IsAsciiHexDigit((char)nextByte))
```

Comment:
Probably still needed for netstandard2.0.  Seems like a good use case for the new static method extensions.